Makefile

```diff
@@ -290,17 +290,24 @@ frontend-fast:
 	rsync -av --delete --delete-excluded --exclude=reports \
 		frontend/build/ lib/streamlit/static/
 
+.PHONY: frontend-lib
+frontend-lib:
+	cd frontend/ ; yarn run buildLib;
+
+.PHONY: frontend-app
+frontend-app:
+	cd frontend/ ; yarn run buildApp
+
```

Comment:
Can you add short docstrings for these rules (just single-line descriptions under the `.PHONY` line)